src/veh_type.h

```diff
@@ -140,118 +140,85 @@ struct transform_terrain_data {
     time_duration post_field_age = 0_turns;
 };
 
+const std::vector<std::pair<std::string, std::string>> vpart_variants = {
+    { "cover", "Cover" }, { "cross", "Cross" },
+    { "horizontal_front", "Front Horizontal" },
+    { "horizontal_front_edge", "Front Edge Horizontal" },
+    { "horizontal_rear", "Rear Horizontal" },
+    { "horizontal_rear_edge", "Rear Edge Horizontal" },
+    { "horizontal_2_front", "Front Thick Horizontal" },
+    { "horizontal_2_rear", "Rear Thick Horizontal" },
+    { "ne_edge", "Front Right Corner" }, { "nw_edge", "Front Left Corner" },
+    { "se_edge", "Rear Right Corner" }, { "sw_edge", "Rear Left Corner" },
+    { "vertical_right", "Right Vertical", }, { "vertical_left", "Left Vertical" },
+    { "vertical_2_right", "Right Thick Vertical" },
+    { "vertical_2_left", "Left Thick Vertical" },
+    { "vertical_T_right", "Right T Joint" }, { "vertical_T_left", "Left T Joint" },
+    { "front_left", "Front Left" }, { "front_right", "Front Right" },
+    { "rear_left", "Rear Left" }, { "rear_right", "Rear Right" },
+    // these have to be last to avoid false positives
```

Comment:
If we did do a JSON list these "defaults" could be left hard-coded and not included in the json, and loaded last?

Reply:
I'm not sure I'm smart enough to make that happen if a mod can potentially load a new variant name.